changelog.d/15522.misc

```diff
@@ -0,0 +1 @@
+Remove references to supporting per-user flag for msc2654 (#15522).
```

Comment:
I guess we can't update the old changelog since it has shipped?

```suggestion
Remove references to supporting per-user flag for [MSC2654](https://github.com/matrix-org/matrix-spec-proposals/pull/2654) (#15522).
```

Reply:
The only changelog that referenced MSC2654 is in #15345, which hasn't merged so I've already fixed that. The changelog for the first PR just said "Add an admin API to support per-user feature flags" and didn't mention the specific features involved.